src/config.js

```diff
@@ -155,12 +155,20 @@ export async function discoverConfigFiles({ getHomeDir = os.homedir } = {}) {
     path.join(process.cwd(), 'package.json'),
     // Look for a magic config in the current working directory.
     path.join(process.cwd(), magicConfigName),
+    // Look for a magic hidden config (preceded by dot) the current working directory.
+    path.join(process.cwd(), `.${magicConfigName}`),
   ];
 
   const configs = await Promise.all(
     possibleConfigs.map(async (fileName) => {
       const resolvedFileName = path.resolve(fileName);
       if (await fileExists(resolvedFileName)) {
+        if (path.basename(magicConfigName) === magicConfigName) {
+          log.warn(
+            `Config file "${magicConfigName}" is deprecated. ` +
+              `Please use ".${magicConfigName}"`,
+          );
+        }
```

Comment:
Sorry for not having made clear that adding a warning was a thought I had but not yet something I had touched base with the rest of the team to confirm if we really wanted that.

We discussed about this PR in the triage we did today and we agreed that we can keep both filenames for the project dir in the longer term and so that we don't need nor want to warn here.

Apologies again for not having made clear that the warning was not yet an actual "request for a change" yet in my previous review comment.

Reply:
never mind. just reset it